checker/src/main/java/org/checkerframework/checker/i18nformatter/I18nFormatUtil.java

```diff
@@ -373,7 +373,7 @@ private static final int findKeyword(String s, String[] list) {
 
             // Try trimmed lowercase.
             String ls = s.trim().toLowerCase(Locale.ROOT);
-            if (ls != s) {
+            if (!ls.equals(s)) {
```

Comment:
I don't think this is needed. Both `trim` and `toLowerCase` return the receiver object if nothing needs to change. So comparing for reference equality is enough. Or why would this be needed?
Maybe instead add a comment explaining this.